LinuxMac.sh

```diff
@@ -103,8 +103,8 @@ echo "mamba installed"
             echo "Aborting. You need at least $mssSize GB of space to install mamba and MSS" && exit 1;
     fi &&
     echo $HOME
-    . "$HOME/Miniforge/etc/profile.d/conda.sh" &&
-    . "$HOME/Miniforge/etc/profile.d/mamba.sh" &&
+    . "$HOME/miniforge3/etc/profile.d/conda.sh" &&
+    . "$HOME/miniforge3/etc/profile.d/mamba.sh" &&
```

Comment:
I tried on a fresh user, by 

```
sudo useradd -s /bin/bash -m exampleuser
sudo su - exampleuser
git clone https://github.com/Open-MSS/mss-install.git
cd mss-install/
git checkout miniforge3
./LinuxMac.sh -a

ls ../
 Miniforge   mss-install 
```
there is the old name

Reply:
the changed name comes when the -a option is not used.

Have a look if you can make it consistant.

Reply:
see https://github.com/Open-MSS/mss-install/blob/main/LinuxMac.sh#L85 and other occurence.